### ISSUE TITLE
openssl error in console followed by vs code "The window has crashed"

### ISSUE THREAD

**Issue by DarwinJS** [Microsoft/vscode#37508]
After my windows 10 machine has been running for a while, visual studio code gets in an odd state (whether it has been running or not).

When I launch it from the command line with a folder name target I get the error:

`openssl config failed: error:02001003:system library:fopen:No such process`

Then when Visual Studio Code launches I get:
![image](https://user-images.githubusercontent.com/11597190/32323834-ae5fa18a-bf9f-11e7-8345-eceb4e315c18.png)

No matter if I click Reopen or Close - it does not get out of this state.  If I then launch it from an icon with no directory - I still get the error.

If I use the "-n" option the problem goes away - so I think it has to do with specific files that I have open that it reopens on startup - but I have no way of knowing what they are.

When I use -n I am still getting the above openssl error - so that **might** be unrelated or is in combination with a specific document.

Ok so -n does not always fix it - if I open a folder that only has only two .ps1 and one .nuspec, it crashes.

Also, If I restart my computer it seems to go away until the computer has been running for quite some time again.

**Comment by octref** [comment]
To start off, what's your exact VS Code version? Did this start to happen after a VS Code / OS update?

**Comment by DarwinJS** [comment]
![image](https://user-images.githubusercontent.com/11597190/32372475-62613182-c06b-11e7-8ac3-bea582538d04.png)

It's hard to exactly tell - but if it did start with an update it was probably 17.1 or 17.2

Interesting - I just had a scenario where running in the current folder repeatedly had the problem => launching like this "code ."

Then I tried "code -n ." - same problem.

Then I did "code -n" - launched fine.  Closed and relaunched with "code ." (same folder as previous attempts) and then it worked.  It *seems* like it loaded the same set of previous docs as when it previously falied.

Extensions only shows two - but I know I also have the Settings Sync extension running ( FYI I've seen other discrepancies in the installed plugins list in the past):

![image](https://user-images.githubusercontent.com/11597190/32372583-da7362a8-c06b-11e7-9677-8ea0a757da91.png)
